cmd/erasure-multipart.go

```diff
@@ -283,6 +283,17 @@ func (er erasureObjects) newMultipartUpload(ctx context.Context, bucket string,
 		parityBlocks = er.defaultParityCount
 	}
 
+	ecOrg := parityBlocks
+	for _, disk := range onlineDisks {
+		di, err := disk.DiskInfo(ctx)
```

Comment:
Would this not affect performance here @klauspost - since we are going to lose IOPS here?

Reply:
@harshavardhana That is why there is a TTL cache in place for DiskInfo.

`getLoadBalancedDisks`, `getOnlineDisks`, `getOnlineDisksWithHealing` all use it.

Reply:
yeah but still imagine concurrent I/O we are loosing crazy amount of IOPS here

Reply:
It will never be more than 1 IOPS per disk.

Sure we can use `IsOnline()`, but `(*xlStorage).IsOnline()`  just returns true, so that would need to be modified with bigger risk of regressions.

> `never add it for .minio.sys/`

Isn't config written with n/2 always?

Reply:
No `.minio.sys/` inherit from STORAGE_CLASS default - so it will be EC:4 

> It will never be more than 1 IOPS per disk.

We should test it out and see this doesn't add any further regressions.

Reply:
It will not be worse in either case since it just adds more protection if needed.